scripts/aspects_migration.js

```diff
@@ -0,0 +1,57 @@
+'use strict'
+// utility for reading in the entire curriculum from a file path
+const fs = require('fs')
+const Insight = require('../lib/insight')
+const childProcess = require('child_process').execSync
+
+let curriculumPath
+let topic
+let curriculum
+
+const aspects = ['new', 'workout', 'deep', 'introduction', 'obscura']
+
+function loadCurriculumFromLocalPath () {
+  process.argv.forEach((val, index) => {
+    if (index === 2) {
+      console.log(`${index}: ${val}`)
+      curriculumPath = val
+    } else if (index === 3) {
+      console.log(`${index}:${val}`)
+      topic = val
+    }
+  })
+  curriculum = childProcess(`find ${curriculumPath}/${topic} -name "*.md"`).toString('utf-8')
+  curriculum = curriculum.split('\n')
+  console.log(typeof (curriculum))
+  return curriculum
+}
+
+function migrateAspectsFromTags () {
+  loadCurriculumFromLocalPath()
+    .forEach(insightPath => {
+      if (!insightPath.includes('README.md') && !insightPath.includes('LICENSE.md') && fs.existsSync(insightPath)) {
+        const insightFile = fs.readFileSync(insightPath, 'utf-8')
+        const insight = new Insight({ body: insightFile, path: insightPath })
+        if (insight.metadata.tags !== undefined) {
```

Comment:
I see you call multiple times `insight.metadata.tags`
to be clearer you can do: `const { metadata: {tags }} = insight` and you have you `tags` variable assigned